new-site/src/docs/components/navigation/tabs.mdx

```diff
@@ -20,6 +20,10 @@ import PageTabs from '../../../components/PageTabs';
   A navigation component is the main way for users to navigate and locate themselves when using a service. It includes some key features present in most services to keep user experience consistent between them.
 </LeadParagraph>
 
+<Notification label="A visual rework is coming soon!" className="siteNotification">
+  The HDS team will be unifying the Navigation component with the Hel.fi project navigation during 2022.
```

Comment:
Should we omit the year and use some more vague term, for example, "in the near future"?